source/aimk

```diff
@@ -1809,6 +1809,9 @@ if ( $JNI == 1 ) then
          case lx-ia64: # debian squeeze; no openjdk in wheezy, but has gcj
          case lx-s390: # debian squeeze; wheezy has s390/cacao/libjvm.so
          case lx-s390x:         # no openjdk in debian wheezy; has gcj
+         case lx-loong64:
+	     set JAVA_LIB_ARCH = loong64
+             breaksw
```

Comment:
This breaks the build for ia64, s390, and s390x.

These cases are supposed to fall through to the lines that are under `lx-ppc64le`:

    set JAVA_LIB_ARCH = `expr substr $buildarch 4 9`
    breaksw

but by adding the `lx-loong64` case in this location, they now set JAVA_LIB_ARCH to loong64, which is obviously wrong for those architectures...